app/actors/hyku_addons/actors/creator_institutional_email_actor.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+module HykuAddons
+  module Actors
+    class CreatorInstitutionalEmailActor < Hyrax::Actors::AbstractActor
+      def create(env)
+        check_creator_institutional_email(env) && next_actor.create(env)
+      end
+
+      def update(env)
+        check_creator_institutional_email(env) && next_actor.update(env)
+      end
+
+      private
+
+        def check_creator_institutional_email(env)
+          creator_hash = JSON.parse(env.attributes[:creator].try(:first) || "{}")
+
+          return [] if creator_hash.blank?
+
+          creators = creator_hash.map do |creator|
+            return creator if creator["creator_name_type"] == "Organizational"
```

Comment:
Don't forget to add a break return after your guard statements to clearly illustrate what is happening.